test/xapi/track_test.rb

```diff
@@ -2,6 +2,16 @@
 require_relative '../../lib/xapi'
 
 class TrackTest < Minitest::Test
+  def assert_archive_contains(filenames, zip)
+    files = []
+    Zip::InputStream.open(zip) do |io|
+      while (entry = io.get_next_entry)
+        files << entry.name
+      end
+    end
+    assert_equal filenames.sort, files.sort
+  end
```

Comment:
I have some suggestions about the naming here: `assert_archive_contains(filenames, zip)`

What do you think about:

`assert_archive_filenames(zip, filenames)`
* 'contains' can be ambiguous and it's not clear whether the method will check all or only some of the filenames, and if `contains` refers to the data that is zipped up.
* The arguments to the method are in the same order as they are in the method name.